day_7/10_little_monkeys.js

```diff
@@ -0,0 +1,42 @@
+// 10 Little Monkeys
+
+
+
+for (var i = 10; i >= 1; i--){
+  if (i > 1){
+    console.log(i + " little monkeys jumping on the bed, One fell off and bumped his head, Mama called the doctor and the doctor said, 'No more monkeys jumping on the bed!'");
+  } else if (i == 1) {
+    console.log(i + " little monkey jumping on the bed, He fell off and bumped his head, Mama called the doctor and the doctor said, 'Get those monkeys right to bed!'");
+   }
+};
+
+
+
+// Bonus
+
+function anyNumberOfMonkeys(start, finish) {
+  for (var i = start; i >= finish; i--){
+    if (i > 1){
+      console.log(i + " little monkeys jumping on the bed, One fell off and bumped his head, Mama called the doctor and the doctor said, 'No more monkeys jumping on the bed!'");
+    } else if (i == 1) {
+      console.log(i + " little monkey jumping on the bed, He fell off and bumped his head, Mama called the doctor and the doctor said, 'Get those monkeys right to bed!'");
+     }
+  }
+};
+
+console.log(anyNumberOfMonkeys(101, 99));
+
+
+// Double Bonus?
+
+function bottlesOfBeer(start, finish) {
```

Comment:
Love it! Killed it this time through! Great code style and indentation throughout